modules/display.js

```diff
@@ -0,0 +1,16 @@
+export const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/zRPcLNnykoeas7KKzkpU/scores';
+
+export const mainDynamicBox = document.querySelector('.dynamic-box');
+// const gameID = 'zRPcLNnykoeas7KKzkpU';
+
+export default async function displayData() {
```

Comment:
- [ ] Please convert this function to using the arrow function instead in order to practice the new ES6 features. 😄 🙏